homeassistant/components/reolink/strings.json

```
@@ -857,6 +857,9 @@
       },
       "packing_time": {
         "name": "Recording packing time"
+      },
+      "post_rec_time": {
+        "name": "Post recording time"
```

This needs a hyphen, otherwise "post" is read as a verb:

```suggestion
        "name": "Post-recording time"
```